src/backend/windows/application.rs

```diff
@@ -51,17 +54,21 @@ pub(crate) struct Application {
 struct State {
     quitting: bool,
     windows: HashSet<HWND>,
+    main_thread_cb_queue: (SharedEnqueuer<MainThreadCb>, SharedDequeuer<MainThreadCb>),
 }
 
 /// Used to ensure the window class is registered only once per process.
 static WINDOW_CLASS_REGISTERED: AtomicBool = AtomicBool::new(false);
 
+static MSG_RUN_MAIN_CB_QUEUE: AtomicU32 = AtomicU32::new(0);
```

Comment:
I think there's a better way to write this type with the recently-stabilized-but-not-yet-released-as-stable `std::sync::OnceCell` and, later, its lazy init equivalent. But it is not the future yet. 🤷

Reply:
It looks like we already depend on the `once_cell` crate, if you want to use that in the meantime.